src/screens/DomainScreen/elements/Header/elements/DomainFollowerNumber.js

```diff
@@ -1,41 +1,40 @@
 import * as React from 'react';
-import { StyleSheet, Text, View } from 'react-native'
+import {StyleSheet, Text, View} from 'react-native';
 
-import { COLORS } from '../../../../../utils/theme';
-import { Gap } from '../../../../../components';
-import { colors } from '../../../../../utils/colors';
-import { fonts, normalize, normalizeFontSize } from '../../../../../utils/fonts';
-import { getSingularOrPluralText } from '../../../../../utils/string/StringUtils';
+import {COLORS} from '../../../../../utils/theme';
+import {Gap} from '../../../../../components';
+import {fonts, normalize, normalizeFontSize} from '../../../../../utils/fonts';
+import {getSingularOrPluralText} from '../../../../../utils/string/StringUtils';
 
 const DomainFollowerNumber = ({followers}) => {
-    return (
-        <View style={styles.container}>
-            <Text style={styles.followersNumber}>{followers}</Text>
-            <Gap width={normalize(4)} />
-            <Text style={styles.followersText}>
-                {getSingularOrPluralText(followers, 'Follower', 'Followers')}
-            </Text>
-        </View>
-    )
-}
+  return (
+    <View style={styles.container}>
+      <Text style={styles.followersNumber}>{followers}</Text>
+      <Gap width={normalize(4)} />
+      <Text style={styles.followersText}>
+        {getSingularOrPluralText(followers, 'Follower', 'Followers')}
+      </Text>
+    </View>
+  );
+};
 
 const styles = StyleSheet.create({
-    container: {
-        flexDirection: 'row',
-        marginTop: 6,
-    },
-    followersNumber: {
-        color: '#00ADB5',
-        fontFamily: fonts.inter[700],
-        fontSize: normalizeFontSize(14),
-        lineHeight: normalizeFontSize(17),
-    },
-    followersText: {
-        color: COLORS.black,
-        fontFamily: fonts.inter[400],
-        fontSize: normalizeFontSize(14),
-        lineHeight: normalizeFontSize(17),
-    },
-})
+  container: {
+    flexDirection: 'row',
+    marginTop: 6
+  },
+  followersNumber: {
+    color: COLORS.holyTosca,
+    fontFamily: fonts.inter[700],
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(17)
+  },
+  followersText: {
+    color: COLORS.black,
+    fontFamily: fonts.inter[400],
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(17)
+  }
+});
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's no newline at the end of the file. It's a common convention to end files with a newline because some tools might have issues processing the last line if it doesn't end with a newline.
```diff
- \ No newline at end of file
+ 
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->